src/quo/components/notifications/toast/view.cljs

```diff
@@ -77,16 +76,19 @@
     :or   {type :neutral icon :i/placeholder}}]
   (let [context-theme (or theme (quo.theme/get-theme))
         icon-name     (case type
-                        :positive :i/correct
-                        :negative :i/incorrect
+                        :positive (if (= theme :light)
+                                    :i/correct
+                                    :i/correct-dark)
+                        :negative (if (= theme :light)
+                                    :i/incorrect
+                                    :i/incorrect-dark)
```

Comment:
@alwx - The best way to handle colors in icons is using SVG.

https://github.com/status-im/status-mobile/blob/ec2b95c3709d2f5c73b2000689777aaa2bb73421/src/quo/components/icons/svg.cljs#L1

This will help us reduce the clutter of images/icons needed for each theme.

Reply:
Honestly, in the current form it just creates clutter somewhere else — namely in the namespace called `quo.components.icons.svg`. In addition, those icons are very hard to be replaced because instead of putting them in resources we're now writing Clojure code for them. 
I would say I'm gonna skip it for now because it feels wrong to replace an imperfect solution with something that's also imperfect and ugly.

Reply:
Also the discussion is here: https://discord.com/channels/1103692771585433630/1103692773363810317/1195414255038648360